eslint.config.js

```diff
@@ -120,13 +120,6 @@ export default [
 			'prefer-const': 'off' // this turns let foo = $derived into a const otherwise
 		}
 	},
-	{
-		name: 'local/svelte-runes-globals',
-		files: ['**/*.svelte.js', '**/*.svelte.ts', '**/*.svelte.*.js', '**/*.svelte.*.ts'],
-		languageOptions: {
-			parser: svelteParser
-		}
```

Comment:
I think this setting is unnecessary because it’s handled by `@sveltejs/eslint-config`.